drivers/src/memory_layout.rs

```diff
@@ -55,26 +38,15 @@ pub const ROM_NSTACK_ORG: u32 = 0x5001FC00;
 //
 // Memory Sizes In Bytes
 //
+pub const PERSISTENT_DATA_SIZE: u32 = 53 * 1024;
```

Comment:
Can you put a comment explaining what how PERSISTENT_DATA_SIZE is computed?

Reply:
Added a comment. Let me know if any more details are needed.